### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Link a Cluster-Link a Cluster_HD Insight_Screen Reader: Jaws not notifying the user about the error message displayed

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3613]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS1.3.1

**Note: Similar issue is observed in** 
1.Link a Cluster-Link a Cluster_Livy Service

**Repro Steps:**
1)Turn on the screen reader (JAWS)
2)Launch IntelliJ IDEA 
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Fill all the required fields(Incorrect data)
9)Verify whether Jaws is notifying the user about the error message displayed 

**Actual Result:**
Jaws not notifying the user about the error message displayed

**Expected Result:**
Jaws should notify the user about the displayed error message 

**User Impact:**
Visually disabled user will find difficult if Jaws is not notifying the user about the displayed error message

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug _Jaws not notifying the user about the error message displayed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646644/Bug._Jaws.not.notifying.the.user.about.the.error.message.displayed.pptx)

**Comment by t-rufang** [comment]
Note: Screen reader can't read the error message displayed
![image](https://user-images.githubusercontent.com/32627233/65672891-61a0b780-e07c-11e9-8523-b49c4fc705b9.png)

**Comment by wezhang** [comment]
May the error console is not focused?

**Comment by t-rufang** [comment]
We fixed the focus issue in #3679. After the fix, when we failed to link a cluster, the dialog will focus at the error details console view. However, JAWS are not reading the content in the dialog. Notice that we are using `com.intellij.execution.impl.ConsoleViewImpl` which is supported by IntelliJ and it's out of the scope of our toolkit.

**Comment by t-rufang** [comment]
- Importance of experience: Important. JAWS is not reading the content in Authentication error details console view when focus is at the console view.
- Quality of experience: Annoying. Keyboard user has to use `Up` arrow to move the cursor to the first character so that JAWS will start reading the error message.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Alexandria-R** [comment]
Approved to close as external to intelliJ

**Comment by Amulya-95** [comment]
Closing this bug as per the discussion with SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749713
